ros2/src/geoscenario_server/geoscenario_server/geoscenario_server.py

```diff
@@ -67,7 +76,8 @@ def __init__(self):
         
         self.traffic = SimTraffic(self.lanelet_map, self.sim_config)
 
-        gsfiles = self.get_parameter('scenario_files').get_parameter_value().string_array_value
+        gsfiles_param = self.get_parameter('scenario_files').get_parameter_value().string_value
+        gsfiles = [f.strip() for f in gsfiles_param.split(',') if f.strip()]
```

Comment:
that's exactly what `string_array_value` does! Just a comma separated list of file paths. All you gain here is that we don't have to give `[]`.